wazo_calld/plugins/voicemails/resources.py

```diff
@@ -49,10 +56,15 @@ class VoicemailMessageUpdateSchema(Schema):
     folder_id = fields.Integer(required=True)
 
 
+class VoicemailGreetingCopy(Schema):
```

Comment:
The convention is to add `Schema` suffix for schema class

Reply:
done